content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -85,7 +85,7 @@ different Kubernetes components.
 | `CSIMigrationOpenStack` | `true` | Beta | 1.18 | |
 | `CSIMigrationvSphere` | `false` | Beta | 1.19 | |
 | `CSIMigrationPortworx` | `false` | Alpha | 1.23 | |
-| `CSIMigrationRBD` | `false` | Alpha | 1.23 | |
+| `csiMigrationRBD` | `false` | Alpha | 1.23 | |
```

Comment:
Per the [docs style guide](https://kubernetes.io/docs/contribute/style/style-guide/), the current format is accurate and appropriate  since while discussing API objects UpperCamelCase is used.